### PR TITLE
Update ununifid-installation-and-setup.md

### PR DESCRIPTION
**Pull request by glukosseth**
# Description

### Corresponding Issue
**Issue number** - 3

## Entry to the community program
**Discord-userId** - `Aleksei#7906`   
**UnUniFi-walletAddress** - `ununifi1ydtts8a9r5jr0qmls9cy60p2j9ewvg6mrnkh4z`

### Author Checklist
- [v] Checked Contribute guide line document
- [v] provided a link to the relevant issue or specification
- [v] reviewed "Files changed" and left comments if necessary
- [v] entered the necessary information for the community program

### Reviewers Checklist
- [ ] confirmed the content is effective

### CODE REVIEW

**Comment by glukosseth** [comment]
@hikaruNagamine  ok, thanks for your comment, do I need to do something to correct PR?

**Comment by hikaruNagamine** [comment]
@glukosseth 
Please see the comments at this link for the correction.
https://github.com/UnUniFi/gitbook/pull/14#discussion_r911114052
https://github.com/UnUniFi/gitbook/pull/14#discussion_r911121257

File: validators/ununifid-installation-and-setup.md [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+Follow the instructions [here](https://go.dev/doc/install) to install Go.
+For an Ubuntu LTS, you can probably use:
+```Bash
+# Please install Go v1.18 or later
+# from $HOME dir
+$ wget https://golang.org/dl/go1.18.2.linux-amd64.tar.gz
+$ sudo rm -rf /usr/local/go
+$ sudo tar -C /usr/local -xzf go1.18.2.linux-amd64.tar.gz
+$ go version
+go version go1.18.2 linux/amd64
```
Comment (hikaruNagamine) [inline validators/ununifid-installation-and-setup.md:31]:
```suggestion
# Please install Go v1.17
# from $HOME dir
$ wget https://go.dev/dl/go1.17.linux-amd64.tar.gz
$ sudo rm -rf /usr/local/go
$ sudo tar -C /usr/local -xzf go1.17.linux-amd64.tar.gz
$ go version
go version go1.17 linux/amd64
```
I would like the go version to match the 1.17 indicated in the README of the [chain repository](https://github.com/UnUniFi/chain#install).
So please make this correction.

**Review by hikaruNagamine — CHANGES_REQUESTED** [review]
@glukosseth 
There is one more point that we would like you to correct, please review the comments.

**Comment by glukosseth** [comment]
ok, done

**Review by hikaruNagamine — APPROVED** [review]
LGTM

**Comment by hikaruNagamine** [comment]
@taiki1frsy 
I have approved the review.
Please confirm your approval.

Please check here as well.
https://github.com/UnUniFi/gitbook/pull/15
https://github.com/UnUniFi/gitbook/pull/16

**Review by taiki1frsy — APPROVED** [review]

**Pull request merged**